### PR TITLE
Updates for dev

### PR DESCRIPTION
**Pull request by ymmit85**

### CODE REVIEW

File: AsBuiltReport.VMware.vROps.psd1 [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+RootModule = 'AsBuiltReport.VMware.vROPs.psm1'
+
+# Version number of this module.
+ModuleVersion = '1.0'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = '1e32cd0d-e251-4bf6-994d-a14c5ee1115d'
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.psd1:21]: Did you create a new manifest or was this copied? if it was copied, you will need to update the GUID using `Update-ModuleManifest`

File: AsBuiltReport.VMware.vROps.psd1 [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+# Generated on: 24/03/2019
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = 'AsBuiltReport.VMware.vROPs.psm1'
+
+# Version number of this module.
+ModuleVersion = '1.0'
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.psd1]: Should follow [semantic versioning](https://semver.org/)

File: AsBuiltReport.VMware.vROps.psd1 [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = '1e32cd0d-e251-4bf6-994d-a14c5ee1115d'
+
+# Author of this module
+Author = 'Tim Williams'
+
+# Company or vendor of this module
+CompanyName = 'Unknown'
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.psd1]: Comment out `CompanyName`

File: AsBuiltReport.VMware.vROps.psd1 [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+# Copyright statement for this module
+Copyright = '(c) 2019 Tim Williams. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'A PowerShell module to generate an as built report on the configuration of VMware vRealize Operations (vROPs).'
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.psd1]: vROps

File: AsBuiltReport.VMware.vROps.psd1 [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+# FormatsToProcess = @()
+
+# Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+# NestedModules = @()
+
+# Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+FunctionsToExport = 'Invoke-AsBuiltReport.VMware.vROPs'
+
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = '*'
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.psd1]: Comment out

File: AsBuiltReport.VMware.vROps.psd1 [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+# Variables to export from this module
+VariablesToExport = '*'
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.psd1]: Comment out

File: AsBuiltReport.VMware.vROps.psd1 [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+# Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
+AliasesToExport = '*'
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.psd1]: Comment out

File: AsBuiltReport.VMware.vROps.psd1 [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+# List of all files packaged with this module
+# FileList = @()
+
+# Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
+PrivateData = @{
+
+    PSData = @{
+
+        # Tags applied to this module. These help with module discovery in online galleries.
+        # Tags = @()
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.psd1]: Update as follows `Tags = 'AsBuiltReport', 'Report', 'VMware', 'vRealize', 'vROps', 'Documentation', 'PScribo', 'PSEdition_Desktop', 'Windows'`

File: AsBuiltReport.VMware.vROps.psd1 [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+        # A URL to the license for this module.
+        # LicenseUri = ''
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.psd1]: `LicenseUri = 'https://raw.githubusercontent.com/AsBuiltReport/AsBuiltReport.VMware.vROps/master/LICENSE'`

File: AsBuiltReport.VMware.vROps.psd1 [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+
+        # A URL to the main website for this project.
+        # ProjectUri = ''
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.psd1]: `ProjectUri = 'https://github.com/AsBuiltReport/AsBuiltReport.VMware.vROps'`

File: AsBuiltReport.VMware.vROps.psd1 [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+
+        # A URL to an icon representing this module.
+        # IconUri = ''
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.psd1]: `IconUri = 'https://raw.githubusercontent.com/AsBuiltReport/AsBuiltReport/master/AsBuiltReport.png'`

File: AsBuiltReport.VMware.vROps.psd1 [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+
+        # ReleaseNotes of this module
+        # ReleaseNotes = ''
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.psd1]: `ReleaseNotes = 'https://raw.githubusercontent.com/AsBuiltReport/AsBuiltReport.VMware.vROps/master/CHANGELOG.md'`

File: AsBuiltReport.VMware.vROps.psd1 [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+#
+# Generated by: Tim Williams
+#
+# Generated on: 24/03/2019
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = 'AsBuiltReport.VMware.vROPs.psm1'
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.psd1]: vROps

**Review by tpcarman — CHANGES_REQUESTED** [review]

**Review by ymmit85 — COMMENTED** [review]
Updates as requested

**Review by tpcarman — APPROVED** [review]

**Review by tpcarman — APPROVED** [review]

**Review by tpcarman — APPROVED** [review]

**Review by tpcarman — APPROVED** [review]

**Review by tpcarman — APPROVED** [review]

**Comment by tpcarman** [comment]
Resubmit PR

**Pull request closed without merging**